cdk/lib/mparticle-api.ts

```diff
@@ -33,9 +41,28 @@ export class MParticleApi extends GuStack {
 			`/${this.stage}/${this.stack}/${app}/sarResultsBucket`,
 		).stringValue;
 
-		// this must be the same as used in the code
+		const deletionRequestsTopicArn = StringParameter.fromStringParameterName(
+			this,
+			'DeletionRequestsTopicArn',
+			`/${this.stage}/${this.stack}/${app}/deletionRequestsTopicArn`,
+		).stringValue;
+
 		const sarS3BaseKey = 'mparticle-results/';
 
+		const deletionDlq = new Queue(this, `${app}-deletion-dlq`, {
+			queueName: `${app}-deletion-dlq-${this.stage}`,
+			retentionPeriod: Duration.days(14),
+		});
+
+		const deletionRequestsQueue = new Queue(this, `${app}-deletion-queue`, {
+			queueName: `${app}-deletion-queue-${this.stage}`,
```

Comment:
nitpick: although we don't have a standard, I think appending dlq on the end is fine, then we can reuse the name.  (note that you can only search queues by prefix)
https://eu-west-1.console.aws.amazon.com/sqs/v3/home?region=eu-west-1#/queues
```suggestion
    const queueName = `${app}-user-deletions-${this.stage}`;
		const deletionDlq = new Queue(this, `${queueName}-dlq`, {
			queueName: `${app}-${queueNameSuffix}-dlq-${this.stage}`,
			retentionPeriod: Duration.days(14),
		});

		const deletionRequestsQueue = new Queue(this, `${app}-deletion-queue`, {
			queueName,
```